db/migrate/20131118232818_encrypt_miq_database_registration_http_proxy_password_field.rb

```diff
@@ -1,5 +1,7 @@
 class EncryptMiqDatabaseRegistrationHttpProxyPasswordField < ActiveRecord::Migration
-  class MiqDatabase < ActiveRecord::Base; end
+  class MiqDatabase < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
```

Comment:
Yeah, I have to understand why that was needed in the first place.  We definitely had this problem in the past...I'll probably have to do some serious git log spelunking :)

Are you saying I should remove it from this PR?